packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue

```diff
@@ -30,54 +30,78 @@
 					</div>
 					<InputText
 						:disabled="true"
-						class="p-inputtext-sm timespan-list"
+						class="p-inputtext-sm timespan-list mb-2"
 						v-model="requestStepListString"
 					/>
-					<h4
-						class="primary-text green-text"
-						v-if="!showAdditionalOptions"
-						@click="toggleAdditonalOptions"
-					>
-						<i class="pi pi-angle-right" /> Show additional options
-					</h4>
-					<h4
-						class="primary-text green-text"
-						v-if="showAdditionalOptions"
-						@click="toggleAdditonalOptions"
-					>
-						<i class="pi pi-angle-down" /> Hide additional options
-					</h4>
 					<div v-if="showAdditionalOptions">
 						<div class="button-column">
 							<label>Tolerance</label>
-							<tera-input-number
-								:min="0"
-								:max="1"
-								:min-fraction-digits="0"
-								:max-fraction-digits="7"
-								v-model="knobs.tolerance"
-							/>
+							<div
+								class="input-tolerance animate-width animation-ease-in-out animation-duration-225"
+							>
+								<tera-input-number
+									class="w-2"
+									:min="0"
+									:max="1"
+									:min-fraction-digits="0"
+									:max-fraction-digits="7"
+									v-model="knobs.tolerance"
+								/>
+								<Slider
+									v-model="knobs.tolerance"
+									:min="0"
+									:max="1"
+									:step="0.01"
+									class="w-full mr-2"
+								/>
```

Comment:
@ngraham76 

Just to note, sliders are not very useful in my opinion in Terarium, things can be pretty sensitive and needs to be fine-tuned, and at times it is much easier to just enter the number directly.

Reply:
In that case you can do both, a slider and an input field.

Reply:
Yup, we have both - an input field and a slider if someone prefers fine tuning their input with the slider.